src/dbnode/persist/fs/migration/migration_test.go

```diff
@@ -60,9 +60,7 @@ func TestToVersion1_1Run(t *testing.T) {
 		fsOpts.InfoReaderBufferSize(), fsOpts.DecodingOptions(), persist.FileSetFlushType)
 	require.Equal(t, 1, len(results))
 	infoFileResult := results[0]
-	indexFd, err := os.Open(path.Join(fsOpts.FilePathPrefix(), fmt.Sprintf("data/%s/%d/fileset-%d-0-index.db",
-		nsId.String(), shard, infoFileResult.Info.BlockStart)))
-	require.NoError(t, err)
```

Comment:
Think the error check may need to be readded

Reply:
Yeah, I just need to drop the `err` here since I check it in the `openFile` method.